bugbot/rules/assignee_no_login.py

```diff
@@ -142,6 +165,7 @@ def handle_bug(self, bug, data):
             "priority": bug["priority"],
             "severity": bug["severity"],
             "keywords": bug["keywords"],
+            "history": bug["history"],
```

Comment:
Why do we need to keep the history? Could we use it here directly and pass the result instead?

Reply:
Yes we could, but then I would have overwrite/add on top of the functions `bughandler` and `get_bugs`. Is there a major benefit of changing the number of outputs of `handle_bug`?

Reply:
I meant to use the `get_priority_change_date()` here and pass the "priority" value instead of the whole history.

Reply:
I meant to use the `get_priority_change_date()` here and pass `is_old_priority` instead of the whole history.

Reply:
Got it! Like this? 623e88aa27cc8193e1416112a82cf65f5aafac81